integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/wait/service/WaitService.java

```diff
@@ -28,12 +28,13 @@
         while (!timeout && !exit) {
             LOGGER.info("Waiting round {} and elapsed time {} ms", attempts, System.currentTimeMillis() - startTime);
             try {
-                success = statusChecker.checkStatus(t);
+                statusChecker.refresh(t);
             } catch (Exception ex) {
                 LOGGER.debug("Exception occurred during waiting: {}", ex.getMessage(), ex);
```

Comment:
could you please change the error message to `Exception occurred during refresh`